libmachine/cert/cert.go

```diff
@@ -195,12 +207,15 @@ func ValidateCertificate(addr, caCertPath, serverCertPath, serverKeyPath string)
 		return false, ErrValidatingCert{err}
 	}
 
+	log.Debugf("Configuration: +%v", tlsConfig)
+
 	dialer := &net.Dialer{
 		Timeout: time.Second * 2,
 	}
 
 	_, err = tls.DialWithDialer(dialer, "tcp", addr, tlsConfig)
 	if err != nil {
+		log.Debugf("Certs are not valid: %s", err)
```

Comment:
Yes line 216 from this diff above causes that message to be printed on the client and on the server I see that error about cert dates in docker.log on the host.